### PR TITLE
Add Athena toolkits

### PR DESCRIPTION
**Pull request by shuaijiang**

### CODE REVIEW

File: toolkits/athena/extract_meta.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Copyright 2021  Xiaomi Corporation (Author: Yongqing Wang)
```
Comment (chenguoguo) [inline toolkits/athena/extract_meta.py:1]: You might want to add shebang

File: toolkits/athena/prepare_data.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
+#
+#!/usr/bin/env bash
```
Comment (chenguoguo) [inline toolkits/athena/prepare_data.py:2]: This is not bash, is it?

**Review by chenguoguo — CHANGES_REQUESTED** [review]
Generally looks good but I added some comments.

File: toolkits/athena/prepare_data.py (hunk 2 above)
Comment (shuaijiang) [reply toolkits/athena/prepare_data.py:2]: not bash

File: toolkits/athena/extract_meta.py (hunk 1 above)
Comment (shuaijiang) [reply toolkits/athena/extract_meta.py:1]: sorry

**Review by chenguoguo — APPROVED** [review]
Great, merging this1

**Pull request merged**